### PR TITLE
bump rust version

### PR DESCRIPTION
**Pull request by tcoratger**
# Pull Request type

Time spent on this PR: 5 min

Please check the type of change your PR introduces:

- [ ] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, renaming)
- [X] Refactoring (no functional changes, no API changes)
- [ ] Build-related changes
- [ ] Documentation content changes
- [ ] Testing

## What is the new behavior?

Rust version update to `1.76.0`.

## Does this introduce a breaking change?

- [ ] Yes
- [X] No

### CODE REVIEW

**Comment by greged93** [comment]
@tcoratger if you want to bump to the latest stable we can go up to 1.77

**Comment by tcoratger** [comment]
> @tcoratger if you want to bump to the latest stable we can go up to 1.77

@greged93 Yes right, will do the same on rpc repo to be sync.

File: rust-toolchain [hunk 1]
```diff
@@ -1,2 +1,2 @@
 [toolchain]
-channel = "1.76.0"
+channel = "1.77.0"
```
Comment (greged93) [inline rust-toolchain:2]: just missing a line break here

Comment (tcoratger) [inline rust-toolchain:2]:
```suggestion
channel = "1.77.0"

```

Comment (tcoratger) [reply rust-toolchain]: Done

**Review by greged93 — APPROVED** [review]

**Pull request merged**